qa/tasks/ceph_test_case.py

```diff
@@ -52,7 +52,7 @@ def get_ceph_cmd_stdout(self, *args, **kwargs):
                 args = args[0]
             kwargs['args'] = args
         kwargs['stdout'] = kwargs.pop('stdout', StringIO())
-        return self.run_ceph_cmd(**kwargs).stdout.getvalue()
+        return self.run_ceph_cmd(**kwargs).stdout.getvalue().strip()
```

Comment:
I expected some tests to fail and that's what happened - /a/vshankar-2024-03-20_11:21:18-fs-wip-vshankar-testing-20240320.053002-testing-default-smithi/7612280

```
2024-03-21T11:15:00.049 DEBUG:teuthology.orchestra.run.smithi050:> sudo adjust-ulimits ceph-coverage /home/ubuntu/cephtest/archive/coverage timeout 120 ceph --cluster ceph fs ls --id testuser -k /tmp/tmp.LWEgRzo10w
2024-03-21T11:15:00.190 INFO:teuthology.orchestra.run.smithi050.stderr:2024-03-21T11:15:00.189+0000 7ff6cc91b640 -1 auth: error parsing file /tmp/tmp.LWEgRzo10w: cannot parse buffer: Malformed input [buffer:3]
2024-03-21T11:15:00.190 INFO:teuthology.orchestra.run.smithi050.stderr:2024-03-21T11:15:00.189+0000 7ff6cc91b640 -1 auth: failed to load /tmp/tmp.LWEgRzo10w: (5) Input/output error
2024-03-21T11:15:00.190 INFO:teuthology.orchestra.run.smithi050.stderr:2024-03-21T11:15:00.189+0000 7ff6cc91b640 -1 auth: error parsing file /tmp/tmp.LWEgRzo10w: cannot parse buffer: Malformed input [buffer:3]
```

Likely the `auth get` before this got effected by the strip.

Reply:
Another instance - /a/vshankar-2024-03-20_11:21:18-fs-wip-vshankar-testing-20240320.053002-testing-default-smithi/7612136

Reply:
Hmm I am very sure I have written a reply on this thread when I pushed fix for this issue but clearly it wasn't posted. Re-writing all I can remember...

Technically, these jobs failed not due to this PR but because it exposed an already existing bug. Ceph commands hang indefinitely when path to keyring is passed and the keyring doesn't have newline character at the end of keyring file. I've written a patch that works around this issue for now. I'll create a ticket for this and investigate it.

Reply:
@rishabh-d-dave do you have a tracker for the above issue you mention?